Mathlib/NumberTheory/Divisors.lean

```diff
@@ -568,9 +568,12 @@ lemma mem_divisorsAntidiag :
     norm_cast
     aesop
   | (n : ℕ), (negSucc x, (y : ℕ)) => by
-    simp [divisorsAntidiag]
+    simp [divisorsAntidiag, eq_comm, negSucc_eq]
```

Comment:
non-terminal simp?

Reply:
It was already non-terminal, blame @YaelDillies. :-) I'll clean it up.

Reply:
Ugh. It results in a 6 line `simp only`.

Reply:
That's why I hadn't squeezed it 🙂

Reply:
For future reference, the previous proof was `simp [...]; norm_cast; aesop` so the simp call was only followed by flexible tactics